terraform/ec2/win/variable.tf

```diff
@@ -53,7 +53,7 @@ variable "test_name" {
 
 variable "test_dir" {
   type    = string
-  default = "./test/feature/windows"
+  default = ""
```

Comment:
Can we change this to `../../../test/feature/windows` as a temporary fix for now?

Reply:
Lemme do that for both win & mac in a follow up so that we can revert easily once we know the root cause.

Reply:
Actually this will only fix the win feature test and not the win restat test... since the default can only be one of those two.
Mac already has the correct path it seems.